locations/spiders/blakes_lotaburgers.py

```diff
@@ -0,0 +1,46 @@
+from scrapy import Spider
+
+from locations.categories import Extras, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class BlakesLotaburgersSpider(Spider):
+    name = "blakes_lotaburgers"
+    item_attributes = {"brand": "Blakes Lotaburgers", "brand_wikidata": "Q492430"}
+    allowed_domains = ["www.lotaburger.com"]
+    start_urls = ["https://www.lotaburger.com/wp-json/ip/v1/blakes_location_json/"]
+
+    def parse(self, response):
+        for data in response.json():
+            item = DictParser.parse(data)
+            item["ref"] = data["post_id"]
+            item["name"] = data["business_name"]
+            item["lat"] = data["coords"]["lat"]
+            item["lon"] = data["coords"]["long"]
+            item["addr_full"] = data["address_pretty"]
```

Comment:
Instead of using `address_pretty` to set `addr_full`, can you set `street_address` to `business_address`? The full address isn't as useful to data consumers as the street address.

Reply:
Sure!